index.js

```diff
@@ -22,14 +22,11 @@ function resolveBuildCommandParts() {
   if (HAPPO_STORYBOOK_BUILD_COMMAND) {
     return HAPPO_STORYBOOK_BUILD_COMMAND.split(' ');
   }
+
+  let version;
+
   try {
-    const version = getStorybookVersionFromPackageJson();
-    if (version === 6) {
-      return ['build-storybook'];
-    }
-    if (version === 7) {
-      return getStorybook7BuildCommandParts();
-    }
+    version = getStorybookVersionFromPackageJson();
```

Comment:
We maybe don't even need storybook version detection with this release, since we'll only be supporting a single version (v9)